runtime/contributing/building_from_source.md

```diff
@@ -11,8 +12,8 @@ chapter).
 
 ## Cloning the Repository
 
-> Deno uses submodules, so you must remember to clone using
-> `--recurse-submodules`.
+:::note Deno uses submodules, so you must remember to clone using
+`--recurse-submodules`. :::
```

Comment:
Convention around the container syntax expects this to be defined via newlines. That way you can extend this to pass additional attributes like custom titles which we don't support at the moment (but might in the future).

```md
:::note title="foo bar"

Some content

:::
```

```suggestion
:::note

Deno uses submodules, so you must remember to clone using
`--recurse-submodules`.

:::
```